src/cookies.rs

```diff
@@ -6,12 +6,12 @@ use webdriver::command::WebDriverCommand;
 use crate::client::Client;
 use crate::error;
 
-/// Type alias for a `cookie::Cookie`
+/// Type alias for a [cookie::Cookie]
```

Comment:
You can actually write
```
[`cookie::Cookie`]
```
and the intra-doc links feature will still pick it up!